src/sql/azdata.proposed.d.ts

```diff
@@ -29,6 +29,10 @@ declare module 'azdata' {
 		export function getConnection(uri: string): Thenable<ConnectionProfile>;
 	}
 
+	export namespace aria {
+		export function alert(msg: string, disableRepeat?: boolean): void;
+		export function status(msg: string, disableRepeat?: boolean): void;
```

Comment:
It would be useful to have comments on how aria alerts and statuses behave _somewhere_, though I'm not sure where they're most likely to be surfaced.